ams/server/src/main/java/com/netease/arctic/server/dashboard/ServerTableDescriptor.java

```diff
@@ -1,225 +1,82 @@
 package com.netease.arctic.server.dashboard;
 
-import com.netease.arctic.data.DataFileType;
-import com.netease.arctic.data.FileNameRules;
-import com.netease.arctic.op.SnapshotSummary;
+import com.netease.arctic.AmoroTable;
+import com.netease.arctic.ams.api.TableFormat;
 import com.netease.arctic.server.dashboard.model.DDLInfo;
 import com.netease.arctic.server.dashboard.model.PartitionBaseInfo;
 import com.netease.arctic.server.dashboard.model.PartitionFileBaseInfo;
+import com.netease.arctic.server.dashboard.model.ServerTableMeta;
 import com.netease.arctic.server.dashboard.model.TransactionsOfTable;
 import com.netease.arctic.server.optimizing.OptimizingProcessMeta;
 import com.netease.arctic.server.optimizing.OptimizingTaskMeta;
 import com.netease.arctic.server.persistence.PersistentBase;
 import com.netease.arctic.server.persistence.mapper.OptimizingMapper;
-import com.netease.arctic.server.persistence.mapper.TableMetaMapper;
 import com.netease.arctic.server.table.ServerTableIdentifier;
 import com.netease.arctic.server.table.TableService;
-import com.netease.arctic.table.ArcticTable;
-import com.netease.arctic.utils.ManifestEntryFields;
 import org.apache.commons.collections.CollectionUtils;
-import org.apache.iceberg.DataFile;
-import org.apache.iceberg.DataOperations;
-import org.apache.iceberg.HasTableOperations;
-import org.apache.iceberg.HistoryEntry;
-import org.apache.iceberg.MetadataTableType;
-import org.apache.iceberg.MetadataTableUtils;
-import org.apache.iceberg.PartitionSpec;
-import org.apache.iceberg.Snapshot;
-import org.apache.iceberg.Table;
-import org.apache.iceberg.TableMetadataParser;
-import org.apache.iceberg.data.GenericRecord;
-import org.apache.iceberg.data.IcebergGenerics;
-import org.apache.iceberg.data.InternalRecordWrapper;
-import org.apache.iceberg.data.Record;
-import org.apache.iceberg.expressions.Expressions;
-import org.apache.iceberg.io.CloseableIterable;
-import org.apache.iceberg.util.PropertyUtil;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
-import java.util.ArrayList;
 import java.util.Collections;
 import java.util.HashMap;
-import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
-import java.util.Set;
 import java.util.stream.Collectors;
 
 public class ServerTableDescriptor extends PersistentBase {
 
-  private static final Logger LOG = LoggerFactory.getLogger(ServerTableDescriptor.class);
+  private final Map<TableFormat, FormatTableDescriptor> formatDescriptorMap = new HashMap<>();
 
   private final TableService tableService;
 
+  {
+    FormatTableDescriptor[] formatTableDescriptors = new FormatTableDescriptor[] {
```

Comment:
Are there any specific benefits to having initialization blocks here? If not, I still suggest moving them to the construct method.